### PR TITLE
#90 lead author review

### PR DESCRIPTION
**Pull request by masizuehlke**
ready for lead author review

### CODE REVIEW

**Comment by masizuehlke** [comment]
Regardless it's not done at github, offline article was reviewed by 10 colleagues of different profiles (HR, MD, HCU, engineers of diff seniority and more than one location).

File: articles/team-fit.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+authorName: Marko Simic
+authorGithubUsername: masizuehlke
+issue: 90
+title: Team Fit
```
Comment (abeggchr) [inline articles/team-fit.md]: Proposals:
- harmonic collaboration with team fit
- 8 characteristics of team fit
- the perfect team fit
-

File: articles/team-fit.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Science tends to measure and quantify things. By making them concrete, tangible, it sets boundaries to itself. But passion breaks them. That’s why they work so well together and why they can’t “be” without each other. For engineer to perform beyond ordinary, an organization must provide a system that will foster the passion. That system is culture.
+
+Culture is group phenomenon. It manifests through shared values and behaviors and experienced through the norms and expectations of a group. When in harmony with personal traits, provides environment that empowers people to perform at their best. Degree of that harmony is called _team fit_.
+
+Software engineering is highly collaborative process. From its very beginning, in discovery phase, until its retirement we’re in constant contact with various stakeholders. Furthermore, as we take on more responsibility, collaboration become more important.
+
+Software is everywhere. Professional services takes us to diverse industry sectors. Creating solution that will meet end-user expectations, require from us to really understand the business, regardless of the role.
```
Comment (abeggchr) [inline articles/team-fit.md]: I'd remove this paragraph.

File: articles/team-fit.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- **Empathy:** Only if you’re honestly concerned and understand others’ feelings and perspective you’ll be able to establish a mutual bond. That bond is fundamental for establishing a feeling of psychological safety in the team – base of every high-performance team.
+- **Adaptability:** We live in the world of diversities. People we work with come from different cultures, opinions and professional/education backgrounds. Projects we work on are from different industries ran by various technologies. As an integral part of these processes, we absorb, analyze and assimilate a huge amount of information and emotions. We adapt.
+- **Pro-active:** In the time of innovations, products do not only respond on impulses from market in real-time, but also try to predict them and act ahead of time. Engineers follow that rhythm, and assisted by proven practices, methodologies and right technology choices bring this to reality.
```
Comment (abeggchr) [inline articles/team-fit.md]: Engineers follow that rhythm, and assisted by proven practices, methodologies and right technology choices bring this to reality.
->
Engineers follow that rhythm and bring this to reality.

File: articles/team-fit.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- **Courage:** It takes courage to challenge the status quo and going out of comfort zone. Those are first two steps on the way forward – road between opportunity and success. 
+- **Endurance:** _Good things take time_. Just try to remember how many things, that made an impact or you’re proud of, took short time to happen.
```
Comment (abeggchr) [inline articles/team-fit.md]: Just try to remember how many things, that made an impact or you’re proud of, took short time to happen.
->
Just try to remember how many things, that you’re proud of, took short time to happen.
-> or even
How many things, that you’re proud of, took short time to happen?

File: articles/team-fit.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+You may ask yourself how realistic is to expect someone to possess all these competencies.
+From my experience, it is. Difference is only which of these are closer to surface.
```
Comment (abeggchr) [inline articles/team-fit.md]: Difference is only which of these are closer to surface.
-> I don't understand exactly, do you mean:
The difference is only to which degree.

File: articles/team-fit.md (hunk 1 above)
Comment (masizuehlke) [reply articles/team-fit.md]: Thank you for you feedback. 
Thinking about your proposals, thought took me do this direction:

- Yourself in world of measurable
- Two sides of complete professional 
- Moving as one
- Can we swim together?
- Are we good for you?
....(and still thinking :-))

File: articles/team-fit.md (hunk 2 above)
Comment (masizuehlke) [reply articles/team-fit.md]: I'm fighting with this one.
You're the third reviewer who said that introduction could be shorter.
And, believe it or not this is almost already a half of original version :)
Reason why it's so extensive is that I wanted to give a context from which I analyzed competences. I'm afraid If I shorten it more, it'll become list without background.

Let me think....

File: articles/team-fit.md (hunk 5 above)
Comment (masizuehlke) [reply articles/team-fit.md]: Yes, it's more clear. "closer to surface" is a metaphor we use in Serbian language.

**Pull request merged**